addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -228,6 +290,28 @@ export class AnalyticDistribution extends Component {
         return options;
     }
 
+    async onSearchMore(request) {
```

Comment:
instead of the "`request`", it could be better to to give the domain you build above instead of building it twice?